rust/chains/hyperlane-ethereum/src/singleton_signer.rs

```diff
@@ -62,13 +63,38 @@ impl SingletonSigner {
     pub fn new(inner: Signers) -> (Self, SingletonSignerHandle) {
         let (tx, rx) = mpsc::unbounded_channel::<SignTask>();
         let address = inner.eth_address();
-        (Self { inner, rx }, SingletonSignerHandle { address, tx })
+        (
+            Self {
+                inner,
+                rx,
+                retries: 5,
+            },
+            SingletonSignerHandle { address, tx },
+        )
+    }
+
+    /// Change default (5) retries for signing
+    pub fn config_retries(&mut self, retries: usize) {
```

Comment:
Why have an unused setter?

Reply:
wanted to make it configurable while not changing the usage
idk why I chose to do it this way lol